lib/bundler/templates/newgem/CODE_OF_CONDUCT.md.tt

```diff
@@ -23,7 +23,7 @@ include:
 Examples of unacceptable behavior by participants include:
 
 * The use of sexualized language or imagery and unwelcome sexual attention or
-advances
+  advances
```

Comment:
Why does this need 2 spaces?

Reply:
It needs two spaces because it's a continuation of the line above. Otherwise, some markdown implementation might interpret it as two separate lists (separated by the `advances` line). See these warnings:
```
CODE_OF_CONDUCT.md:25: MD032 Lists should be surrounded by blank lines
CODE_OF_CONDUCT.md:27: MD032 Lists should be surrounded by blank lines
```
Check a few lines below (the last two bullet points) - they have the two spaces as well.